aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -227,7 +266,11 @@ class QConvPackWeightInt8 final {
         stride.size() == 2,
         "quantized::conv2d_prepack (qnnpack): 2D convolution only");
     TORCH_CHECK(
-        padding.size() == 2,
+        input_padding.size() == 2,
+        "quantized::conv2d_prepack (qnnpack): Specify top/left padding only. "
```

Comment:
Can we update the error message to specify input/output padding (here and below)?